modules/dataframe/tests/benchmarks/join.cpp

```diff
@@ -65,7 +64,10 @@ std::vector<int> left_ = [&]() {
     return left;
 }();
 
-std::vector<int> right_ = [&]() {
+const std::vector<int> right_ = []() {
+    std::random_device rd;
+    std::mt19937 gen(rd());
```

Comment:
doesn't this now give exactly the same sequence as for left_?

Reply:
No, one does a shuffle the other uses distrib?